packages/jupyter-chat/src/model.ts

```diff
@@ -677,6 +698,7 @@ export abstract class AbstractChatModel implements IChatModel {
   private _id: string | undefined;
   private _name: string = '';
   private _config: IConfig;
+  private _readyDelegate: PromiseDelegate<void>;
```

Comment:
```suggestion
  private _readyDelegate = new PromiseDelegate<void>();
```
Not sure it needs to be set in the constructor.

Reply:
Yes, the constructor doesn’t need to do anything for it.